tmt/steps/provision/artemis.py

```diff
@@ -508,9 +511,12 @@ def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecor
                 help=f'A factor for exponential API retry backoff, '
                      f'{DEFAULT_RETRY_BACKOFF_FACTOR} by default.',
                 ),
-            ] + super().options(how)
+            ]
+        return options
```

Comment:
Changes to `options()` seem to be not needed, actually undoing changes made by previous patches.